modules/azure-workload-identity-federation-openshift.adoc

```diff
@@ -18,18 +18,27 @@ For more information, see link:https://docs.openshift.com/container-platform/{oc
 .Procedure
 * To add the federated identity credentials to a user-assigned managed identity, run the following command, for example:
 +
+[IMPORTANT]
+====
+When setting up the delegated scanning, set the subject to *system:serviceaccount:stackrox:sensor*.
+====
++
 [source,terminal]
 ----
 $ az identity federated-credential create \
     --name "${FEDERATED_CREDENTIAL_NAME}" \
-    --identity-name "${MANAGED_IDENTITY_NAME}" \// <1>
+    --identity-name "${MANAGED_IDENTITY_NAME}" \
     --resource-group "${RESOURCE_GROUP}" \
-    --issuer "${OIDC_ISSUER_URL}" \// <2>
-    --subject system:serviceaccount:stackrox:central \ <3>
+    --issuer "${OIDC_ISSUER_URL}" \
+    --subject system:serviceaccount:stackrox:central \
     --audience openshift
 ----
-<1> The managed identity must have all the permissions for federation.
-<2> The issuer must match the service account token issuer of the {ocp} cluster.
-<3> For delegated scanning, set the subject to *system:serviceaccount:stackrox:sensor*.
 +
-For more information about how to configure short-lived tokens by using Microsoft Entra ID managed identities, see link:https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust-user-assigned-managed-identity?pivots=identity-wif-mi-methods-azp[Configure a user-assigned managed identity to trust an external identity provider] (Microsoft Azure documentation).
+where:
++
+--
+`--identity-name`:: Specifies that the managed identity must have all the permissions for federation.
```

Comment:
I don't think this is accurate. It should be "Specifies the identity name. The managed identity must have...."